### PR TITLE
Refactor project structure

### PR DESCRIPTION
**Pull request by brettgoss**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+
+function loadSecrets() {
```
Comment (tweeres04) [inline src/utils.js]: Is this function necessary? I'm assuming your app will already crash if it tries to do something with a non-existent environment variable

File: src/notify.js [hunk 2]
```diff
@@ -1,41 +1,27 @@
...
+    url: secrets.zendeskURL,
+    email: secrets.zendeskEmail,
+    token: secrets.zendeskToken
 });
 
-app.use(bodyParser.json({ extended: false }));
-app.post("/count", (req, res) => {
-    res.status(200);
-    res.send();
+function handlePost(){
```
Comment (tweeres04) [inline src/notify.js]: Could promisify this:

```javascript
function countNewTickets() {
    const queryString = 'query=type:ticket status:new assignee:none group:none';
    return zendesk.search.list(queryString).then(results => {
        var ticketCount = results.length;
        console.log(`Ticket count: ${ticketCount}/${THRESHOLD}`);
        return ticketCount;
    });
}

function handlePost() {
    countNewTickets().then(ticketCount => {
        if (shouldNotify(ticketCount)) {
            sendSlackMessage(ticketCount);
        } else {
            console.log('No message sent');
        }
    });
}

```

Comment (tweeres04) [reply src/notify.js]: Or you could use `async/await` if your environment supports it

Comment (brettgoss) [reply src/notify.js]: Oh, this is much nicer. Thanks!

**Pull request merged**